src/Explorer/Panes/SettingsPane/SettingsPane.tsx

```diff
@@ -116,6 +124,18 @@ export const SettingsPane: FunctionComponent = () => {
     { key: "true", text: "JSON" },
   ];
 
+  const priorityLevelOptionList: IChoiceGroupOption[] = [
+    { key: Constants.PriorityLevel.Low, text: "Low" },
```

Comment:
This is missing the None option,

Reply:
We do not require automatic end to end tests, but you should test this manually yourself before submitting for review. You need to test all cases... all possible options for the priority level.